eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -5,9 +5,9 @@ jobs:
     pool:
       vmImage: vs2017-win2016
     steps:
-      - powershell: |
+      - pwsh: |
           echo "##vso[build.addbuildtag]Scheduled"
-        displayName: 'Tag scheduled builds'
```

Comment:
For my information is the conversion to double quotes here just part of your tooling?  I ask because https://github.com/Azure/azure-sdk-for-net/blob/master/eng/.prettierrc.yml seems to indicate single quotes.

Reply:
We need to pick single or double quotes and standardize across repos.  In this case `eng` has a `.prettierrc.yml` which specifies single quotes, but the files under `eng/pipelines` mostly used double quotes, so I added another `.prettierrc.yml` under `eng/pipelines` to use double quotes for that folder.  JS repo also uses double quotes for the pipeline YML files.

Reply:
I don't have a strong opinion about which option to use but I would suggest just updating the prettierrc.yml file in the eng directory instead of adding another file with a different convention in eng/pipelines.

Reply:
I will create a separate PR to cleanup the prettier config and align everything under `eng` to use either single-quote or double-quote.  Right now I think there's a mix of both quote styles across different folders, so whichever we choose will create a huge diff.

Reply:
Will be fixed in a separate PR.  Tracking issue: https://github.com/Azure/azure-sdk/issues/528